server/src/views/profile.js

```diff
@@ -109,12 +110,23 @@ exports.Profile = class Profile extends React.Component {
     this.setState({ nickname: e.target.value });
   }
 
+  onClickDeleteEverything(e) {
+    e.preventDefault();
+    let ok = confirm("Are you sure you want to permanently delete everything you have created using PageShot?");
+    if (ok) {
+      events.deleteEverything();
+      helpers.request("POST", "/delete");
+      window.location = "/";
```

Comment:
We should probably redirect to something like `/?accountDeleted` and then do a check for that and pop up something on the homepage (which we can do separately from this commit)